conductor/src/main/java/com/bluelinelabs/conductor/changehandler/SharedElementTransitionChangeHandler.java

```diff
@@ -17,17 +17,20 @@
 
 import com.bluelinelabs.conductor.Controller;
 import com.bluelinelabs.conductor.ControllerChangeHandler;
-import com.bluelinelabs.conductor.internal.TransitionUtils;
+import com.bluelinelabs.conductor.internal.LegacyTransitionUtils;
 
 import java.util.ArrayList;
 import java.util.List;
 
 /**
  * A TransitionChangeHandler that facilitates using different Transitions for the entering view, the exiting view,
  * and shared elements between the two.
+ *
+ * @deprecated See {@link TransitionChangeHandler}
```

Comment:
I don't think this link is going to work correctly. Maybe update it with the package name?

Reply:
Works since it's in the same package.

(To clarify, this is referencing the existing `TransitionChangeHandler`, I just didn't want to duplicate the deprecation message.)

Reply:
Oh, whoops. I misread this and thought you were linking to the androidx version.